src/libraries/System.Private.CoreLib/src/System/Globalization/CalendarData.Icu.cs

```diff
@@ -450,7 +456,7 @@ private static unsafe void EnumCalendarInfoCallback(string calendarString, IntPt
             }
         }
 
-        private struct EnumCalendarsData
+        public struct IcuEnumCalendarsData
```

Comment:
```suggestion
        private struct IcuEnumCalendarsData
```
?

I do not see why it had to change to public.

Reply:
It seems like this was when renaming the member I accidentally changed it to public.